README.md

```diff
@@ -11,23 +11,12 @@
 
 _For the installation instructions, [click here](#installation)._
 
-NNCF provides a suite of advanced algorithms for Neural Networks inference optimization in [OpenVINO&trade;](https://github.com/openvinotoolkit/openvino) with minimal accuracy drop.
+NNCF provides a suite of advanced algorithms for reducing inference time of Neural Networks executed via [OpenVINO&trade;](https://github.com/openvinotoolkit/openvino) or ONNXRuntime utilizing [OpenVINOExecutionProvider](https://onnxruntime.ai/docs/execution-providers/OpenVINO-ExecutionProvider.html) with attention to models accuracy.
```

Comment:
NNCF does not reduce inference time itself. Also, optimization can include weights size reduction.

Reply:
I think that all NNCF algorithms aim to speed up the inference time of a model - is this the goal of the tool? The important point in the statement is that 
`executed via [OpenVINO&trade;](https://github.com/openvinotoolkit/openvino) or ONNXRuntime utilizing [OpenVINOExecutionProvider](https://onnxruntime.ai/docs/execution-providers/OpenVINO-ExecutionProvider.html)`
Also, can we declare that the size reduction is a consequence of the algorithm?

Reply:
I agree with Aleksei that we should state explicitly the inference time reduction effect that NNCF is actually used for.

But the updated sentence here got too long in my opinion:

```suggestion
NNCF provides a suite of advanced algorithms for reducing inference time of Neural Networks executed via [OpenVINO&trade;](https://github.com/openvinotoolkit/openvino) or [ONNXRuntime](https://onnxruntime.ai/docs/execution-providers/OpenVINO-ExecutionProvider.html) with attention to models accuracy.
```

Reply:
@KodiaqQ - the sentence is not written as "NNCF provides a suite of advanced algorithms for reducing inference time of Neural Networks". It is written as "NNCF provides a suite of advanced algorithms for reducing inference time of Neural Networks **executed via OpenVINO**". There is no contradiction with what you are saying neither in current wording nor in the extended version proposed by @kshpv. 

However, I would not put ONNXRuntime on the same level here in this sentence as OpenVINO. It is ultimately OpenVINO that we are targeting, be it through IE directly or via ONNXRuntime. 
`... via OpenVINO (either directly in [Inference Engine](link) or by using [ONNXRuntime OpenVINOExecutionProvider](link))`

Reply:
As you wish, guys. I didn't agree that NNCF algorithms affect **inference time**  directly while executing. _Bias Correction-like_ algorithms does not affect it, also as _Channel Alignment_, for example.

Reply:
Bias-Correction is not the algorithm that we provide to the user, it is the a part of Quantization, isn't it?

Reply:
I wouldn't like to keep talking about this case. I would resolve the original conversation.